README.md

```diff
@@ -76,30 +85,36 @@ apl
 <br> example: sof-boot-once.sh reboot
 <br> Effect: when system boots up it will auto reboot again
 
-* sof-dmesg-check.sh
-<br> Check dmesg for errors
-<br> Contains keyword lists to ensure we're stopping due to a real error.
-
-* tplgtool.py
-<br> tplgtool dump info from tplg binary file.
+* sof-combinatoric.py
+<br> Used to compute permutations or combinations of the various pipelines
+     avilable during tests if multiple are needed at once.
 
-* sof-tplgreader.py
-<br> tplgtool.py wrapper, it read info from tplgtool.py to analyze tplgs.
+* sof-disk-usage.sh
+<br> Check current disk size for avoid system problem without enough space
```

Comment:
This line needs some improvement, change to `Used to make sure we have enough disk space to collect logs and avoid system problems`?